cmd/image/upload/upload.go

```diff
@@ -2,15 +2,163 @@ package upload
 
 import (
 	"fmt"
+	"github.com/ppc64le-cloud/pvsadm/pkg"
+	"github.com/ppc64le-cloud/pvsadm/pkg/client"
+	"github.com/ppc64le-cloud/pvsadm/pkg/client/s3utils"
 	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+)
+
+const (
+	AuthEndpoint             = "https://iam.cloud.ibm.com/identity/token"
+	ServiceType              = "cloud-object-storage"
+	UseExistingPromptMessage = "Would You Like to use Available COS Instance for creating bucket?"
+	CreatePromptMessage      = "Would you like to create new COS Instance?"
+	ResourceGroupAPIRegion   = "global"
 )
 
 var Cmd = &cobra.Command{
 	Use:   "upload",
 	Short: "Upload the image to the IBM COS",
-	Long:  `Upload the image to the IBM COS`,
+	Long: `Upload the image to the IBM COS
+pvsadm image upload --help for information
+
+# Set the API key or feed the --api-key commandline argument
+export IBMCLOUD_API_KEY=<IBM_CLOUD_API_KEY>
+
+Examples:
+
+# using InstanceName
+pvsadm image upload --bucket bucket0711 -i myfile.txt --instance-name pvsadm-cos-instance
```

Comment:
Have more meaningful names in the -i option probably some image name like `rhcos-461.ova.gz`